### ISSUE TITLE
Usability issues with text-only browser zoom

### ISSUE THREAD

**Issue by quincylvania** [openstreetmap/iD#7965]
Most browser support [increasing the text size](https://support.mozilla.org/en-US/kb/accessibility-features-firefox-make-firefox-and-we#w_text-zoom) of the page while leaving the size of images and other elements alone. This is a useful accessibility feature, but fails if text containers have a fixed size and the larger text overflows.

We need to audit iD for instances where this happens, since there are clearly some issues…

<img width="913" alt="Screen Shot 2020-09-09 at 2 19 30 PM" src="https://user-images.githubusercontent.com/2046746/92638597-7fc0f100-f2a8-11ea-95e5-3fed815cc771.png">

Re: #7713

**Comment by quincylvania** [comment]
Spent a lot of energy finessing the UI for this, and I'm pretty happy with the result. It's not perfect but it's gone from unusable to usable.

<img width="1151" alt="Screen Shot 2020-09-11 at 1 07 59 PM" src="https://user-images.githubusercontent.com/2046746/92953554-e7ee0f00-f42f-11ea-9e0a-1a5743b0ca67.png">
